src/xenia/kernel/xnotifylistener.h

```diff
@@ -30,13 +30,8 @@ union XNotificationKey {
   };
   XNotificationID id;
 
-  static constexpr XNotificationID get_id(uint8_t mask_index,
-                                          uint16_t local_id) {
-    XNotificationKey key = {};
-    key.mask_index = mask_index;
-    key.local_id = local_id;
-    return key.id;
-  }
+  constexpr XNotificationKey(XNotificationID notification_id)
+      : id(notification_id){};
```

Comment:
For consistency with other parts of Xenia (except for `reenter_exception`…), please add a space between the `)` and the `{` and remove the `;`.

Reply:
Also, this structure requires a proper `XNotificationID` for creation now. Please add:
```cpp
constexpr XNotificationKey() : id(XNotificationID(0)) {}
```
so it can still be build manually as a `struct` by assigning values to the bit fields (but zeroing by default is still needed so all unset bits are zeroed).

Alternatively, `id` can have a default value of `= XNotificationID(0)` specified (and the constructor without arguments should be `= default`), but in this case, as far I know, `id` must be the first field, and that's pretty easy to forget (see my message with the text "Zero-initialized to clear irrelevant bits" in #dev on the Discord server).

On the other hand though, it's better to have `id` as the first field that is `= 0` by default so we can have other constructors which would also have all unused bits zeroed (because fields are initialized in the order they're declared in the class — so if you have both `id(0)` and `local_id(something)` in the initializer list, with the current code, `local_id` will be initialized first, and will be erased by `id` zeroing).